docs/build/support.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Support
+description: A comprehensive guide on how to reach out for support
+---
+
+import ColoredText from '/src/components/ColoredText';
+
+# Support
+
+This section explains how you can get support and stay up-to-date with Celo developer news.
+
+:::tip
+Help us improve by sharing your feedback on missing tools, unclear sections, or any challenges you encounter by filling out this [Feedback Form](https://forms.gle/14t1bhQyGDm7YQNG8).
+:::
```

Comment:
are we commited to checking this doc? can we create a monthly task for this in linear? otherwise, I would propose, we should just recommend to create an issue.